openpype/pipeline/create/creator_plugins.py

```diff
@@ -208,10 +204,33 @@ def __init__(
         # - we may use UI inside processing this attribute should be checked
         self.headless = headless
 
-        self.apply_settings(project_settings, system_settings)
+        expect_system_settings = False
+        if is_func_signature_supported(
+            self.apply_settings, project_settings
+        ):
+            self.apply_settings(project_settings)
+        else:
+            expect_system_settings = True
+            # Backwards compatibility for system settings
+            self.apply_settings(project_settings, system_settings)
+
+        init_overriden = self.__class__.__init__ is not BaseCreator.__init__
+        if init_overriden or expect_system_settings:
```

Comment:
Ok, so this warning is now always logged for all classes inheriting from the new `Creator` due to the `Creator` class already overriding `__init__`. :)